package.json

```diff
@@ -23,13 +23,14 @@
     "gen:types": "tsc --build",
     "prepare": "tsc --build",
     "test:web": "playwright-test test/**/*.spec.js --cov && nyc report",
-    "test:node": "c8 --check-coverage --branches 95 --functions 83 --lines 94 mocha test/**/*.spec.js",
+    "test:node": "c8 --check-coverage --branches 95 --functions 83 --lines 94 node --test test/**/*.spec.js",
```

Comment:
Does changing the test framework need to happen in this PR?

Reply:
switched back to mocha